src/services/todo.services/todoServices.ts

```diff
@@ -27,9 +29,9 @@ export function createToDo(title: string, description: string): ToDo {
   return toDo;
 }
 
-export default function getToDo(id: string): ToDo {
+export default function getToDo(id: string): ToDo | undefined {
```

Comment:
every service method could use `neverthrow` types. 
Let's talk about it in our daily so I can show you the example :)